learn/functions/lambda.md

```diff
@@ -3,32 +3,49 @@
 
 **Lambda functions** in Python are **anonymous** functions that can be defined and declared **inline** with the rest of the code. 
 
-## What is a lambda function in Python
+## What is a Lambda Function in Python
 
-Lambda functions take in parameters and return a value like a regular Python function, but are **usually only used once** and therefore not given a name. Instead, they are **passed as an argument to other functions** or used in an expression.
+In Python, a lambda function is a concise way to create small, anonymous functions. Unlike regular functions defined with the `def` keyword, lambda functions are created using the `lambda` keyword and are typically used for short, immediate operations.
 
-### Example 1: Defining a lambda function to multiply two numbers
+The syntax for a lambda function is simple:
 
-```python 
-multiply = lambda x, y: x * y
-
-# calling the lambda function
-print(multiply(5, 3)) # output: 15
+```python
+lambda arguments: expression
 ```
 
-### Example 2: Using a lambda function as an argument to the map() function
+Here, `lambda` is the keyword, `arguments` are the function's input parameters, and `expression` is the operation the function performs.
 
+Here are also some examples of declaration and usage:
+
+1. **Basic Addition**
```

Comment:
Let's convert these items to H3 headings. But don't use bold highlighting `**` inside the heading tags.

Reply:
done